app/controllers/application_controller.rb

```diff
@@ -382,10 +373,6 @@ def prompt_to_verify_sp_required_mfa
     redirect_to sp_required_mfa_verification_url
   end
 
-  def prompt_to_setup_non_restricted_mfa
-    redirect_to login_additional_mfa_required_url
```

Comment:
Are we expecting this `additional_mfa_required` route to go away altogether? I still see a handful of references to it (views, routes in `routes.rb`, etc).

- `app/views/users/additional_mfa_required/show.html.erb`
- `app/views/users/mfa_selection/index.html.erb` (maybe?)
   - Is `MfaSelectionController` related to this and should be removed as well?

Reply:
Mfa selection controller is not related to this so shouldnt be removed.